upgrade/README.md

```diff
@@ -56,7 +56,48 @@ The upgrade is a guided process starting with [Upgrade Management Nodes and CSM
 - Although it is not recommended, the [Booting CSM `barebones` image](../operations/validate_csm_health.md#5-booting-csm-barebones-image)
   test may be skipped if all compute nodes are active running application workloads.
 
-See [Validate CSM Health](../operations/validate_csm_health.md).
+1. (`ncn-m002#`) If a typescript session is already running in the shell, then first stop it with the `exit` command.
+
+1. (`ncn-m002#`) Start a typescript.
+
+    ```bash
+    script -af /root/csm_upgrade.$(date +%Y%m%d_%H%M%S).post_upgrade_health_validation.txt
+    export PS1='\u@\H \D{%Y-%m-%d} \t \w # '
+    ```
+
+    If additional shells are opened during this procedure, then record those with typescripts as well. When resuming a procedure
+    after a break, always be sure that a typescript is running before proceeding.
+
+1. Validate CSM health.
+
+    See [Validate CSM Health](../operations/validate_csm_health.md).
+
+1. (`ncn-m002#`) Stop typescripts.
+
+    Stop any typescripts that were started during the health validation procedure.
```

Comment:
I am assuming the user already knows how to stop this. Same comment for the rest of the places below where we mention "Stop any typescripts"

Reply:
Yeah, I went back and forth on that one. I would be fine explicitly including the exit command.